spring-ai-alibaba-graph/spring-ai-alibaba-graph-core/pom.xml

```diff
@@ -166,6 +165,20 @@
             <optional>true</optional>
         </dependency>
 
+        <dependency>
+            <groupId>io.modelcontextprotocol.sdk</groupId>
+            <artifactId>mcp</artifactId>
+            <version>0.10.0</version>
+            <optional>true</optional>
```

Comment:
Here should be a compile dependency. There's no `starter` for graph-core to manage dependencies, so it should make all it's necessary dependencies transitive.